### PR TITLE
Added /skull command

### PR DESCRIPTION
**Pull request by mathias-github**
Not quite like Bukkit's Essentials, since you can't have custom skulls in you inventory (yet), but it works.

### CODE REVIEW

File: moderator.lua [hunk 1]
```diff
@@ -209,6 +209,29 @@ function HandleTPSCommand(Split, Player)
...
-
-
-
+function HandleSkullCommand(Split, Player)                                                                                                                              
+	if Split[2] == nil then                                                                                                                                         
+		Player:SendMessageInfo("Usage: "..Split[1].." <username>")                                                                                            
+		return true                                                                                                                                             
+	end                                                                                                                                                             
+                                                                                                                                                                        
+	pos = GetPlayerLookPos(Player)
```
Comment (NiLSPACE) [inline moderator.lua]:
```lua
local pos = ...

local LookingAtHead = ...
```

Comment (mathias-github) [inline moderator.lua]: Thanks for pointing out! When should and shouldn't I specify local?

Comment (SphinxC0re) [inline moderator.lua]: Temporarily used variables should always be local, otherwise you get a memory overflow

**Comment by mathias-github** [comment]
Should be rebased now.

File: moderator.lua [hunk 2]
```diff
@@ -187,7 +187,7 @@ function HandleAntiOchCommand(Split,Player)
 end
 
 --- Handles console tps command, wrapper to HandleTpsCommand function
---  Necessary due to MCServer now supplying additional parameters
```
Comment (SphinxC0re) [inline moderator.lua]: Did you force-push the rebased repo? Because I already changed this before

**Comment by mathias-github** [comment]
Don't merge until I've rebased this properly.

**Comment by mathias-github** [comment]
Alright, looks correct now.

**Comment by tonibm19** [comment]
I'll merge then. Im not really sure about what does that command do but the code looks well for me.
Thanks.

**Comment by tonibm19** [comment]
Well it seems I have lost writte access to Essentials D:

**Comment by SphinxC0re** [comment]
No, you didn't :smile:

**Comment by mathias-github** [comment]
Whenever you look at a skull and execute the command /skull <username>, the skullowner changes to the specified username (the skin of the skull changes to the same skin as the username).

**Comment by tonibm19** [comment]
The merge pull request button seems to have disappeared for me.

**Comment by mathias-github** [comment]
Bump?

**Comment by tonibm19** [comment]
As i said i cant merge anything.
@madmaxoft
@tigerw
@NiLSpace

**Comment by NiLSPACE** [comment]
Can you change things now @tonibm19?

**Comment by tonibm19** [comment]
Yep, thanks

**Pull request merged**